packages/exo/src/exo-tools.js

```diff
@@ -24,41 +26,131 @@ const { quote: q, Fail } = assert;
 const { apply, ownKeys } = Reflect;
 const { defineProperties, fromEntries } = Object;
 
+/**
+ * A method guard, for inclusion in an interface guard, that does not
+ * enforce any constraints of incoming arguments or return results.
+ */
+const RawMethodGuard = M.call().rest(M.rawValue()).returns(M.rawValue());
+
+const REDACTED_RAW_VALUE = '<redacted rawValue>';
+
 /**
  * A method guard, for inclusion in an interface guard, that enforces only that
  * all arguments are passable and that the result is passable. (In far classes,
- * "any" means any *passable*.) This is the least possible enforcement for a
- * method guard, and is implied by all other method guards.
+ * "any" means any *passable*.) This is the least possible non-raw
+ * enforcement for a method guard, and is implied by all other
+ * non-raw method guards.
  */
-const MinMethodGuard = M.call().rest(M.any()).returns(M.any());
+const PassableMethodGuard = M.call().rest(M.any()).returns(M.any());
 
 /**
  * @param {Passable[]} syncArgs
- * @param {MethodGuardPayload} methodGuardPayload
+ * @param {object} argConfig
+ * @param {number} argConfig.declaredLen
+ * @param {boolean} argConfig.hasRestArgGuard
+ * @param {boolean} argConfig.needRedaction
+ * @param {Pattern} argConfig.paramsPattern
+ * @param {boolean} argConfig.rawRestArgs
+ * @param {number[]} argConfig.redactedIndices
```

Comment:
If you pull this out into a separate ArgConfig (or ArgsConfig or RedactConfig) type, then you could add a 
```
@returns {ArgConfig)
```
to `redactRawValues` below, which I think helps readability.